src/js/common/schemaform/FormStartControls.jsx

```diff
@@ -53,8 +53,8 @@ class FormStartControls extends React.Component {
         <div>
           <ProgressButton
               onButtonClick={this.handleLoadForm}
-              buttonText="Resume previous application"
-              buttonClass="usa-button-primary"/>
+              buttonText="Continue with Application"
+              buttonClass="usa-button-primary title-case"/>
```

Comment:
Is `title-case` an appropriate name for this? It is actually only setting `text-transform` to `none`, but for situations intended for title cased text.